glean-core/rlb/src/lib.rs

```diff
@@ -49,7 +49,8 @@ pub use core_metrics::ClientInfoMetrics;
 pub use glean_core::{
     global_glean,
     metrics::{DistributionData, MemoryUnit, RecordedEvent, TimeUnit},
-    setup_glean, CommonMetricData, Error, ErrorType, Glean, HistogramType, Lifetime, Result,
+    setup_glean, traits, CommonMetricData, Error, ErrorType, Glean, HistogramType, Lifetime,
+    Result,
```

Comment:
Should `global_glean` and `setup_glean` be just `use` instead of `pub use`?

Reply:
Huh, I think we can drop that now from exposing it, you are right.
Let me test that in m-c though to save us from one more unnecessary roundtrip if I overlook something

Reply:
ok, I only grepped for it and now there's no more setup_glean or global_glean left in m-c!